src/qt/pivx/sendconfirmdialog.cpp

```diff
@@ -118,18 +118,22 @@ void TxDetailDialog::setData(WalletModel *model, const QModelIndex &index){
 
 }
 
-void TxDetailDialog::setData(WalletModel *model, WalletModelTransaction &tx){
+void TxDetailDialog::setData(WalletModel *model, WalletModelTransaction &tx) {
     this->model = model;
     this->tx = &tx;
     CAmount txFee = tx.getTransactionFee();
     CAmount totalAmount = tx.getTotalTransactionAmount() + txFee;
 
     ui->textAmount->setText(BitcoinUnits::formatWithUnit(nDisplayUnit, totalAmount, false, BitcoinUnits::separatorAlways) + " (Fee included)");
-    if(tx.getRecipients().size() == 1){
+    if (tx.getRecipients().size() == 1) {
         ui->textSend->setText(tx.getRecipients().at(0).address);
+        ui->textSendLabel->setText(tx.getRecipients().at(0).label);
```

Comment:
performance nit: wouldn't it be better to save the `QList` rather than making three separate calls to `tx.getRecipients()` ?

Reply:
Not sure, you'd have to  additionally create a SendCoinsRecipient object and an int.
But I've changed it anyway...